aws/resource_aws_apigatewayv2_api.go

```diff
@@ -54,6 +54,10 @@ func resourceAwsApiGatewayV2Api() *schema.Resource {
 				Required:     true,
 				ValidateFunc: validation.StringLenBetween(1, 128),
 			},
+			"body": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Can add
```go
				ValidateFunc:     validation.ValidateJsonString,
				DiffSuppressFunc: suppressEquivalentJsonDiffs,
```

Reply:
Hi, as it documents, [You can write OpenAPI definitions in YAML or JSON](https://swagger.io/docs/specification/basic-structure/).

In order to improve the validate and different method here, we might need to implement a new pair of functions for OpenAPI only, which can also be used in `aws/resource_aws_api_gateway_rest_api.go`.

In that case, that field should meet either `StringIsJSON` or `StringIsYAML`. And the same for the  `DiffSuppressFunc `.

Reply:
We might need another merge request to implement those validate and diff functions first for the  `aws/resource_aws_api_gateway_rest_api.go` then reuse it here.

I will try that tomorrow. 😃